bpf/flows.c

```diff
@@ -81,16 +80,7 @@ static inline int flow_monitor(struct __sk_buff *skb, u8 direction) {
         if (pkt.rtt > aggregate_flow->flow_rtt) {
             aggregate_flow->flow_rtt = pkt.rtt;
         }
-
-        long ret = bpf_map_update_elem(&aggregated_flows, &id, aggregate_flow, BPF_ANY);
-        if (trace_messages && ret != 0) {
-            // usually error -16 (-EBUSY) is printed here.
-            // In this case, the flow is dropped, as submitting it to the ringbuffer would cause
-            // a duplicated UNION of flows (two different flows with partial aggregation of the same packets),
-            // which can't be deduplicated.
-            // other possible values https://chromium.googlesource.com/chromiumos/docs/+/master/constants/errnos.md
-            bpf_printk("error updating flow %d\n", ret);
-        }
+        bpf_spin_unlock(&aggregate_flow->lock);
```

Comment:
this is update in place we get pointer to flow in memory so we can update it w/o need to call `bpf_map_update_elem()` like this example https://elixir.bootlin.com/linux/latest/source/tools/testing/selftests/bpf/progs/test_map_lock.c